algorithme-evaluation.md

```diff
@@ -90,9 +93,70 @@ Le modèle est entrainé sur les variables d'apprentissage suivantes:
 
 Voir [ce document](https://github.com/signaux-faibles/opensignauxfaibles/blob/master/js/reduce.algo2/docs/variables.json) pour la définition et la source des champs présents en base.
 
-### Seuils de détection :construction_worker:
+### _Evaluation du modèle: lexique_
 
-Les seuils ont pour l'instant été arbitrairement fixés pour garantir des volumes raisonnables d'entreprises détéctées. Il serait pertinent d'améliorer cette méthode, notamment en trouvant les seuils qui maximisent notre critère d'évaluation.
+Pour notre problème, nous choisissons d'attribuer un score de 1 aux entreprises ayant un risque maximal de défaillance, et 0 aux entreprises ayant un risque minimal de défaillance. En conséquence, nous avons les définitions suivantes:
+
+Défaillance
+: une entrée en procédure collective, ou une dette urssaf maintenue plus de 3 mois
+
+Elément/établissement positif
+: un établissement connaissant effectivement une défaillance à au moins un moment sur les 18 prochains mois
+
+Elément/établissement négatif
+: le contraire d'un établissement positif
+
+Elément/établissement prédit positif
+: un établissement que notre algorithme identifie comme à risque de défaillance (fort ou modéré) sur les 18 prochains mois
+
+Elément/établissement prédit négatif
+: le contraire d'un établissement prédit positif
+
+Elément/établissement faux positif
+: un établissement pour lequel une défaillance est prédite, mais qui ne connaît pas de défaillance effective
+
+Elément/établissement faux négatif
+: un établissement pour lequel aucune défaillance n'est prédite, mais qui connaît effectivement une défaillance
+
+A partir de ces définitions, on dérive les métriques usuelles d'évaluation d'un algorithme d'apprentissage automatique:
+
+Précision
+: la part d'établissements prédits positifs étant effectivement positifs
+
+Rappel
+: la part d'établissements effectivement positifs étant prédits positifs
+
+Score $F_{\beta}$
```

Comment:
Latex is unfortunately not supported in GitHub-flavored markdown https://stackoverflow.com/questions/11256433/how-to-show-math-equations-in-general-githubs-markdownnot-githubs-blog

Reply:
Fixed in https://github.com/signaux-faibles/documentation/pull/45/commits/c48bbb45a9145e4bd6bc78ff68bdc5fa308585aa